404.html

```diff
@@ -1,16 +1,18 @@
-<script src="/assets/js/runtime~main.d5c28086.js"></script>
-<script src="/assets/js/main.2f77ca8b.js"></script>
-</body>
-</html>
\ No newline at end of file
+<body>
+
+<p>The Honeycomb documentation is now hosted at:
+    <a href=”https://honeycomb.ccv.brown.edu/”>
```

Comment:
Something funky happened with your double quote here?
```suggestion
    <a href="https://honeycomb.ccv.brown.edu/">
```

Reply:
I hate it when this happens. 😓 thanks!